deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -51,38 +51,37 @@ serviceMonitors:
 
     ## Endpoints of the selected service to be monitored
     ## endpoints: []
-       ## Name of the endpoint's service port
-       ## Mutually exclusive with targetPort
-       # - port: ""
+    ## Name of the endpoint's service port
+    ## Mutually exclusive with targetPort
+    #    - port: ""
 
-       ## HTTP path to scrape for metrics
-       ##
-       #   path: /metrics
+    ## HTTP path to scrape for metrics
+    ##
+    #      path: /metrics
     endpoints:
       - port: exporter
         path: /mertics
 
     ## Label selector for services to which this ServiceMonitor applies
     ## selector: {}
-       # matchLabels: {}
-       # Eg- matchLabels:
-       #       cas-type: cstor
```

Comment:
Let's keep the syntax at one place and then state the example. matchLabels: {} states the syntax that it is a list and then show the example below.